tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/PhysicalProjectTreeTests.cs

```diff
@@ -7,9 +7,9 @@ public class PhysicalProjectTreeTests
         [Fact]
         public void Constructor_ValueAsTreeService_SetTreeServiceProperty()
         {
-            var projectTreeService = new Lazy<IProjectTreeService>(() => IProjectTreeServiceFactory.Create());
+            var projectTreeService = new Lazy<IProjectTreeService>(IProjectTreeServiceFactory.Create);
             var projectTreeProvider = new Lazy<IProjectTreeProvider>(() => IProjectTreeProviderFactory.Create());
-            var projectTreeStorage = new Lazy<IPhysicalProjectTreeStorage>(() => IPhysicalProjectTreeStorageFactory.Create());
+            var projectTreeStorage = new Lazy<IPhysicalProjectTreeStorage>(IPhysicalProjectTreeStorageFactory.Create);
```

Comment:
Why is line 11 different? (ie, no analyzer warning is reported?)

Reply:
Good question. I'm on a phone right now so can't check but maybe it's ambiguous due to overloads.